v2/tests/modules/cli02/cmd/root.go

```diff
@@ -0,0 +1,85 @@
+// Copyright 2021 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package cmd
+
+import (
+	"fmt"
+	"os"
+
+	"github.com/spf13/cobra"
+
+	homedir "github.com/mitchellh/go-homedir"
+	"github.com/spf13/viper"
```

Comment:
Note, this folder is generated by cobra CLI without change.
I only use it for testing getting go modules

Reply:
I like the idea of making this very simple module/binary that obviously does nothing. It will help make it easier for new readers to understand what this does - even though this is the default generated cobra code, you have to read through it to figure that out.

Having an empty main with an `_` import is functionally equivalent, but much simpler to grok if you have no background on this file.